docker.go

```diff
@@ -770,6 +771,26 @@ func NewDockerProvider(provOpts ...DockerProviderOption) (*DockerProvider, error
 		config:                tcConfig,
 	}
 
+	// log docker server info
+	{
+		type empty struct{}
+		infoMessage := `%v - Connected to docker: 
+  Server Version: %v
+  API Version: %v
+  Operating System: %v
+  Total Memory: %v MB
+`
+
+		info, err := p.client.Info(context.Background())
+		if err != nil {
+			return nil, err
+		}
+
+		p.Logger.Printf(infoMessage, reflect.TypeOf(empty{}).PkgPath(),
```

Comment:
Super NIT: do you think we need the reflect package her, or just hardcode the "github.com/testcontainers/testcontainers-go" string?

Reply:
Well, I would prefer a hardcode for this small thing.

Reply:
Let's use the string here then. I do not thing the package name changes that much 😅

Reply:
Done